web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -239,8 +240,7 @@ public static boolean startModule(Module mod, ServletContext servletContext, boo
 				if (root.getElementsByTagName("dwr").getLength() > 0) {
 					
 					// get the dwr-module.xml file that we're appending our code to
-					File f = new File(realPath + "/WEB-INF/dwr-modules.xml".replace("/", File.separator));
-					
+					File f = Paths.get(realPath,"WEB-INF","dwr-modules.xml").toFile();
```

Comment:
this change is ok, as it does not change the behavior of the code. Since here the replace is used on a string which we control and you correctly cleaned from "/".